### PR TITLE
[types] Move SRA types from @0xproject/connect to @0xproject/types

### PR DESCRIPTION
**Pull request by LogvinovLeon**
## Description

We want to reuse that types in other places incuding 0x-relayer-template and we need them exported.
Those types and SRA related, not connect related and that's why we created a new package.

## Testing instructions

It builds

## Checklist:

*   [x] Prefix PR title with `[WIP]` if necessary.
*   [x] Prefix PR title with bracketed package name(s) corresponding to the changed package(s). For example: `[sol-cov] Fixed bug`.
*   [x] Add tests to cover changes as needed.
*   [x] Update documentation as needed.
*   [x] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/connect/CHANGELOG.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "version": "2.0.3",
+        "changes": [
+            {
+                "note": "Import SRA-related types from @0xproject/sra-types"
+            }
```
Comment (fabioberger) [inline packages/connect/CHANGELOG.json]: PR nr.

File: packages/sra-types/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+## @0xproject/types
```
Comment (fragosti) [inline packages/sra-types/README.md]: Should be `sra-types`, but also why did you decide not to just put these in types? Or just depend on connect in your package?

File: packages/sra-types/.npmignore [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.*
+yarn-error.log
+/scripts/
+/src/
+tsconfig.json
+/lib/monorepo_scripts/
```
Comment (fabioberger) [inline packages/sra-types/.npmignore]: Remove this one, we don't have `monorepo_scripts` anymore in package `lib` dirs.

File: packages/sra-types/README.md (hunk 2 above)
Comment (fabioberger) [inline packages/sra-types/README.md]: `sra-types`

File: packages/connect/src/index.ts [hunk 4]
```diff
@@ -1,23 +1,21 @@
 export { HttpClient } from './http_client';
 export { ordersChannelFactory } from './orders_channel_factory';
-export {
-    Client,
+export { Client, OrdersChannel, OrdersChannelHandler } from './types';
+import {
```
Comment (fragosti) [inline packages/connect/src/index.ts]: So if you want to use any of these types you need to add sra-types now too? Same question as above.

Comment (fragosti) [reply packages/connect/src/index.ts]: Ok I see you answered this in the description. I guess the only question is why new package instead of just `types`

Comment (fragosti) [reply packages/connect/src/index.ts]: And should connect re-export these types for convenience?

File: packages/sra-types/README.md (hunk 2 above)
Comment (LogvinovLeon) [reply packages/sra-types/README.md]: You're right. Moved everything into types

**Comment by LogvinovLeon** [comment]
@fragosti Could you take a look one more time?

**Pull request merged**